src/pages/video-listing/VideoListing.jsx

```diff
@@ -1,7 +1,21 @@
+import { Tab, VideoCard } from "../../components";
 import "./VideoListing.css";
-
+import { TabData, videos } from "./TabData";
 const VideoListing = () => {
-	return <div>VideoListing</div>;
+	return (
+		<div>
+			<div className="tab-container">
+				{TabData.map((item, index) => (
+					<Tab key={index} label={item} />
```

Comment:
Suggestion: Try using something else as a key rather than index itself

Reply:
i wil change it to id when i will implement API calls.